infrastructure/lib/atat-web-api-stack.ts

```diff
@@ -90,12 +98,26 @@ export class AtatWebApiStack extends cdk.Stack {
       method: HttpMethod.GET,
       handlerPath: packageRoot() + "/api/portfolioDrafts/getPortfolioDraft.ts",
     });
-
+    /*
     const submitPortfolioDraft = new ApiDynamoDBFunction(this, "SubmitPortfolioDraft", {
       table: table,
       method: HttpMethod.POST,
       handlerPath: packageRoot() + "/api/portfolioDrafts/submit/submitPortfolioDraft.ts",
     });
+*/
+    // publish to queue
+    const submitPortfolioDraft = new ApiSQSFunction(this, "SubmitPortfolioDraft", {
+      queue: submitQueue,
+      method: HttpMethod.POST,
+      handlerPath: packageRoot() + "/api/portfolioDrafts/submit/publish.ts",
+    });
+    // subscribe to queue, add item to DB
+    const subscribePortfolioDraftRequest = new ApiSQSDynamoDBFunction(this, "subscribePortfolioDraftRequest", {
```

Comment:
hmm... are these types getting too granular? let's make a note to consider this design at some point. fine for now though. good stuff!